source/frontend/src/features/mapSideBar/tabs/SectionField.tsx

```diff
@@ -4,7 +4,7 @@ import { Col, Row } from 'react-bootstrap';
 import styled from 'styled-components';
 
 interface ISectionFieldProps {
-  label: string;
+  label: string | null;
```

Comment:
Is it necessary to use the null? why not simply empty string?

Reply:
Is not necessary. But I would rather have the 'null' to convey the absence of a label.